### PR TITLE
RO-3834-3841-3843

### PR DESCRIPTION
**Pull request by brindakv**
This PR addresses four updates:
 - RO-3834: Add `pdbx_initial_refinement_model` category and related search metadata for starting models
 - RO-3841: Add `rcsb_entry_info.assembly_count` to advanced search menu
 - RO-3843: Add `drugbank_info.brand_names` to advanced search menu
 - RO-3814: Update enumerations and description of `rcsb_entity_source_organism.provenance_source`

@piehld Any idea why the CARD modifications are showing up as new updates?

### CODE REVIEW

File: dictionary_files/base/rcsb_mmcif_ext-core.dic [hunk 1]
```diff
@@ -1866,8 +1872,9 @@ save__rcsb_entity_source_organism.source_type
 # Add definitions from rcsb_mmcif_comp_model_ext-core.dic
 save__rcsb_entity_source_organism.provenance_source
     _item_description.description
-;  A code indicating the provenance of the source organism details for the entity
-
+;  Reference to the provenance of the source organism details for the entity. 
+   Primary data indicates information obtained from the same source as the structural model.
```
Comment (piehld) [inline dictionary_files/base/rcsb_mmcif_ext-core.dic:1876]: Should we add a note here about the plan to remove `PDB Primary Data` in the future when Data API v2 is released?

Likewise, is there a list somewhere of all planned dictionary/schema changes that will need to made for the Data API v2 release? If so, then we should make sure this is added to that list so we don't forget.

File: dictionary_files/dist/rcsb_mmcif_ext.dic [hunk 2]
```diff
@@ -3336,7 +3349,9 @@ save_
 #
 save__rcsb_entity_source_organism.provenance_source
    _item_description.description  
-;  A code indicating the provenance of the source organism details for the entity
+;  Reference to the provenance of the source organism details for the entity. 
+   Primary data indicates information obtained from the same source as the structural model.
```
Comment (piehld) [inline dictionary_files/dist/rcsb_mmcif_ext.dic:3353]: Same questions here.

**Review by piehld — APPROVED** [review]
Thanks @brindakv. Changes LGTM, but had a few comments about the pending removal of the `"PDB Primary Data"` enumeration.

Also, I'm wondering if you should update the minor schema versions defined here (for `polymer_entity`, `bird_chem_comp`, `drugbank`, `ihm_dev`)?: https://github.com/rcsb/py-rcsb_exdb_assets/blob/01ad971ef7e678ae9d4ba953faf8a745bbb00686/config/exdb-config-schema.yml#L1671

**Comment by piehld** [comment]
Also, to address your question about CARD annotations—that's due to me having had only updated the `pdbx_core` schemas with the CARD annotations, not the `pdbx_comp_model_core` schemas, which your PR is now capturing. Thank you for pointing this out and updating them here.

File: dictionary_files/base/rcsb_mmcif_ext-core.dic (hunk 1 above)
Comment (brindakv) [reply dictionary_files/base/rcsb_mmcif_ext-core.dic:1876]: @piehld I would prefer not to add such information to the dictionary. It should be in documented in the Data API v2 JIRA ticket.

**Comment by brindakv** [comment]
> Thanks @brindakv. Changes LGTM, but had a few comments about the pending removal of the `"PDB Primary Data"` enumeration.
> 
> Also, I'm wondering if you should update the minor schema versions defined here (for `polymer_entity`, `bird_chem_comp`, `drugbank`, `ihm_dev`)?:
> 
> https://github.com/rcsb/py-rcsb_exdb_assets/blob/01ad971ef7e678ae9d4ba953faf8a745bbb00686/config/exdb-config-schema.yml#L1671

Thanks @piehld. I updated all affected schema versions.

File: dictionary_files/base/rcsb_mmcif_ext-core.dic (hunk 1 above)
Comment (piehld) [reply dictionary_files/base/rcsb_mmcif_ext-core.dic:1876]: OK, I understand. I guess we just need to make sure an item about this is added to https://rcsbpdb.atlassian.net/browse/RO-2578

**Comment by piehld** [comment]
Thanks @brindakv, LGTM!

**Pull request merged**